fronts-client/config/webpack.config.dev.js

```diff
@@ -1,7 +1,7 @@
-import path from 'path';
-import common from './webpack.config.common.js';
+const path = require('path');
+const common = require('./webpack.config.common.js');
```

Comment:
We should not be mixing and matching import styles here, so I've moved all webpack config to CommonJS.